packages/experiments/src/slots/examples/Slots.Content.Example.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Button, Text, VerticalStack } from '@uifabric/experiments';
+import { Spinner } from 'office-ui-fabric-react';
+import { stackProps } from './SlotExampleUtils';
```

Comment:
probably should do the full package import paths even in experiments so we don't forget to convert these when it is moved to OUFR. Besides, the reason we do these full package imports in OUFR applies here too :)

Reply:
What is that reason? Can we document import standards somewhere? I can never keep all of the reasons straight particularly when I thought treeshaking eliminated some of them. What is the reason to keep doing full package imports for examples?

Reply:
So you can actually copy these examples into CRA and it will work, for instance.

Reply:
I actually thought Lint-imports is supposed to catch this but apparently not...

Reply:
Is it possible to fix CRA to support this rather than have varying import standards?

Reply:
Guys, the reason has much less to do with CRA, but it certainly does have much to do with writing good examples.

When we import external things for example, those things do not show up in the demo page, so the details of what "stackProps" is is totally hidden to the consumer of the example.

Best practice would be to just avoid importing anything beyond the package based things. 

Alternatively, if there were some way to ensure you can see the content of SlotExampleUtils.ts from the example page, that works too, but i'm not sure how that'd look. Or if it would even make a good example.

Reply:
Just to be clear, my mention of CRA was a "for instance". Thanks David.

Reply:
I'm actually more confused now. Can you please cite actual code lines of what you're looking for?

I understand David's point about not having "helpers" but to have everything self-contained. Therefore, `stackProps` should be present in the file.

But I think Cliff was asking for something different, and I'm still not quite sure what.

Reply:
Are lines 1-3 ok as-is?

Reply:
Yes 1-3 are fine.